app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,111 @@
+$(document).ready(function() {
+  var flag = true;
```

Comment:
No rush right now, but I think it would be pretty cool if you guys refactored this later on to use the `this` keyword. I would add it as an issue to your github repo.